### PR TITLE
3008

### PR DESCRIPTION
**Pull request by lingawakad**
## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/sdr-enthusiasts/plane-alert-db/pulls) to support the maintainers.

### CODE REVIEW

**Comment by Phaeton** [comment]
@dziban303 You seeing an error here, or referring to the failed action run I kicked of last week?

**Comment by Phaeton** [comment]
@lingawakad tag me or dzi with a review request when you are done making changes.

**Comment by lingawakad** [comment]
@Phaeton any time, i'm at a good stopping point for now

File: plane-alert-db.csv [hunk 1]
```diff
@@ -14185,3 +14185,16 @@ E90E07,CX-MIA,Uruguay Police,Robinson R44,R44,Pol,Police Squad,Policia,Copper Ch
...
+AB2764,N818DE,Mt Gilead Full Gospel International Ministries,Cessna 650 Citation,C650,Civ,Jesus He Knows Me,Church Of Private Jets,Send NO Money,Jesus he Knows me,https://www.mtgileadfgim.org/
+A2F4E5,N29UB,JDI Holdings,MIG-29UB,MG29,Civ,Too Much Money,You Can't See Me,Aaaaaaaand It's Gone,Distinctive,https://w.wiki/3ADz
+A52C68,N432VA,SF2VA,Cirrus Jet Vision SF50,SF50,Civ,Safe Return,CAPS,Backpack Tadpole V-Jet,Distinctive,https://w.wiki/B3L6
+A2B1D5,N273CB,Private,Beechcraft T-34,T34P,Civ,I Am Old,Retired Life,Former USAF,Distinctive,https://w.wiki/B6WK
+A67C72,N517LH,Labcorp,Pilatus PC-24,PC24,Civ,Genomes,Medical,Bizjet,As Seen on TV,https://w.wiki/Ahr2
+33FD0A,MM62329,Italian Air Force,Gulfstream 5,GLF5,Mil,VIP,Must Be Nice,Virtute Siderum Tenus,Other Air Forces,https://w.wiki/B6Wo
+AE74D4,22-03441,United States Army,Boeing AH-64E Apache Guardian,H64,Mil,Air Cavalry,Chopper,Attack Helos,Toy Soldiers,https://w.wiki/8k4B
+A0AB51,N142NG,Five Star Marketing,Diamond DA-42 Guardian,DA42,Civ,Surveillance,Covert,Spyplane,Oxcart,https://tinyurl.com/3yvyvhw4
```
Comment (Phaeton) [inline plane-alert-db.csv:14200]: why is this tagged as a Oxcart/spy plane?

File: plane-alert-db.csv [hunk 2]
```diff
@@ -14185,3 +14185,16 @@ E90E07,CX-MIA,Uruguay Police,Robinson R44,R44,Pol,Police Squad,Policia,Copper Ch
...
 E940FB,FAB-002,Gov of Bolivia,Dassault Falcon 50,FA50,Gov,Government,Must Be Nice,Free And Fair Elections,Dictator Alert,https://w.wiki/B3LJ
+AB3966,N822MK,Merck Group,Gulfstream G650,G650,Civ,Pharmaceuticals,Vaccines,Bizjet,As Seen on TV,https://w.wiki/B53f
+AE2FD6,168076,United States Navy,Grumman E-2C Hawkeye,E2,Mil,Eye In The Sky,AWACS,Airborne Early Warning,Oxcart,https://w.wiki/6Su9
+A725DF,N56PJ,Luca Aviation,Cirrus Jet Vision SF50,SF50,Civ,Safe Return,CAPS,Backpack Tadpole V-Jet,Distinctive,https://w.wiki/B3L6
+A00538,N100FG,University Athletic Association,Embraer Phenom 300,E55p,Civ,Egghead Eight,NCAA Div III,Sports?,As Seen on TV,https://w.wiki/B64P
+A062E6,N124DN,Aviation Specialty Services,Cessna 337 Super Skymaster,C337,Civ,I Am Old,Beach Life,Former USAF,Distinctive,https://w.wiki/B64a
+AB2764,N818DE,Mt Gilead Full Gospel International Ministries,Cessna 650 Citation,C650,Civ,Jesus He Knows Me,Church Of Private Jets,Send NO Money,Jesus he Knows me,https://www.mtgileadfgim.org/
+A2F4E5,N29UB,JDI Holdings,MIG-29UB,MG29,Civ,Too Much Money,You Can't See Me,Aaaaaaaand It's Gone,Distinctive,https://w.wiki/3ADz
```
Comment (Phaeton) [inline plane-alert-db.csv:14194]: I think this would be better as a Zoomies versus Distinctive.

File: plane-alert-db.csv [hunk 3]
```diff
@@ -14185,3 +14185,16 @@ E90E07,CX-MIA,Uruguay Police,Robinson R44,R44,Pol,Police Squad,Policia,Copper Ch
 E90E0E,CX-MIE,Uruguay Police,Robinson R66,R66,Pol,Police Squad,Policia,Copper Chopper,Police Forces,https://w.wiki/B3LK
 E940FA,FAB-001,Gov of Bolivia,Dassault Falcon 900EX,F900,Gov,Government,Must Be Nice,Free And Fair Elections,Dictator Alert,https://w.wiki/B3LJ
 E940FB,FAB-002,Gov of Bolivia,Dassault Falcon 50,FA50,Gov,Government,Must Be Nice,Free And Fair Elections,Dictator Alert,https://w.wiki/B3LJ
+AB3966,N822MK,Merck Group,Gulfstream G650,G650,Civ,Pharmaceuticals,Vaccines,Bizjet,As Seen on TV,https://w.wiki/B53f
+AE2FD6,168076,United States Navy,Grumman E-2C Hawkeye,E2,Mil,Eye In The Sky,AWACS,Airborne Early Warning,Oxcart,https://w.wiki/6Su9
+A725DF,N56PJ,Luca Aviation,Cirrus Jet Vision SF50,SF50,Civ,Safe Return,CAPS,Backpack Tadpole V-Jet,Distinctive,https://w.wiki/B3L6
+A00538,N100FG,University Athletic Association,Embraer Phenom 300,E55p,Civ,Egghead Eight,NCAA Div III,Sports?,As Seen on TV,https://w.wiki/B64P
+A062E6,N124DN,Aviation Specialty Services,Cessna 337 Super Skymaster,C337,Civ,I Am Old,Beach Life,Former USAF,Distinctive,https://w.wiki/B64a
```
Comment (Phaeton) [inline plane-alert-db.csv:14192]: I'd set this to Historic or Joe Cool

File: plane-alert-db.csv [hunk 4]
```diff
@@ -14185,3 +14185,16 @@ E90E07,CX-MIA,Uruguay Police,Robinson R44,R44,Pol,Police Squad,Policia,Copper Ch
 E90E0E,CX-MIE,Uruguay Police,Robinson R66,R66,Pol,Police Squad,Policia,Copper Chopper,Police Forces,https://w.wiki/B3LK
 E940FA,FAB-001,Gov of Bolivia,Dassault Falcon 900EX,F900,Gov,Government,Must Be Nice,Free And Fair Elections,Dictator Alert,https://w.wiki/B3LJ
 E940FB,FAB-002,Gov of Bolivia,Dassault Falcon 50,FA50,Gov,Government,Must Be Nice,Free And Fair Elections,Dictator Alert,https://w.wiki/B3LJ
+AB3966,N822MK,Merck Group,Gulfstream G650,G650,Civ,Pharmaceuticals,Vaccines,Bizjet,As Seen on TV,https://w.wiki/B53f
+AE2FD6,168076,United States Navy,Grumman E-2C Hawkeye,E2,Mil,Eye In The Sky,AWACS,Airborne Early Warning,Oxcart,https://w.wiki/6Su9
+A725DF,N56PJ,Luca Aviation,Cirrus Jet Vision SF50,SF50,Civ,Safe Return,CAPS,Backpack Tadpole V-Jet,Distinctive,https://w.wiki/B3L6
+A00538,N100FG,University Athletic Association,Embraer Phenom 300,E55p,Civ,Egghead Eight,NCAA Div III,Sports?,As Seen on TV,https://w.wiki/B64P
```
Comment (Phaeton) [inline plane-alert-db.csv:14191]:
```suggestion
A00538,N100FG,University Athletic Association,Embraer Phenom 300,E55P,Civ,Egghead Eight,NCAA Div III,Sports?,As Seen on TV,https://w.wiki/B64P
```

File: plane-alert-db.csv [hunk 5]
```diff
@@ -14185,3 +14185,16 @@ E90E07,CX-MIA,Uruguay Police,Robinson R44,R44,Pol,Police Squad,Policia,Copper Ch
 E90E0E,CX-MIE,Uruguay Police,Robinson R66,R66,Pol,Police Squad,Policia,Copper Chopper,Police Forces,https://w.wiki/B3LK
 E940FA,FAB-001,Gov of Bolivia,Dassault Falcon 900EX,F900,Gov,Government,Must Be Nice,Free And Fair Elections,Dictator Alert,https://w.wiki/B3LJ
 E940FB,FAB-002,Gov of Bolivia,Dassault Falcon 50,FA50,Gov,Government,Must Be Nice,Free And Fair Elections,Dictator Alert,https://w.wiki/B3LJ
+AB3966,N822MK,Merck Group,Gulfstream G650,G650,Civ,Pharmaceuticals,Vaccines,Bizjet,As Seen on TV,https://w.wiki/B53f
+AE2FD6,168076,United States Navy,Grumman E-2C Hawkeye,E2,Mil,Eye In The Sky,AWACS,Airborne Early Warning,Oxcart,https://w.wiki/6Su9
+A725DF,N56PJ,Luca Aviation,Cirrus Jet Vision SF50,SF50,Civ,Safe Return,CAPS,Backpack Tadpole V-Jet,Distinctive,https://w.wiki/B3L6
```
Comment (Phaeton) [inline plane-alert-db.csv:14190]: I'd use Joe Cool or something other than Distinctive

File: plane-alert-db.csv [hunk 6]
```diff
@@ -14185,3 +14185,16 @@ E90E07,CX-MIA,Uruguay Police,Robinson R44,R44,Pol,Police Squad,Policia,Copper Ch
...
+A00538,N100FG,University Athletic Association,Embraer Phenom 300,E55p,Civ,Egghead Eight,NCAA Div III,Sports?,As Seen on TV,https://w.wiki/B64P
+A062E6,N124DN,Aviation Specialty Services,Cessna 337 Super Skymaster,C337,Civ,I Am Old,Beach Life,Former USAF,Distinctive,https://w.wiki/B64a
+AB2764,N818DE,Mt Gilead Full Gospel International Ministries,Cessna 650 Citation,C650,Civ,Jesus He Knows Me,Church Of Private Jets,Send NO Money,Jesus he Knows me,https://www.mtgileadfgim.org/
+A2F4E5,N29UB,JDI Holdings,MIG-29UB,MG29,Civ,Too Much Money,You Can't See Me,Aaaaaaaand It's Gone,Distinctive,https://w.wiki/3ADz
+A52C68,N432VA,SF2VA,Cirrus Jet Vision SF50,SF50,Civ,Safe Return,CAPS,Backpack Tadpole V-Jet,Distinctive,https://w.wiki/B3L6
+A2B1D5,N273CB,Private,Beechcraft T-34,T34P,Civ,I Am Old,Retired Life,Former USAF,Distinctive,https://w.wiki/B6WK
```
Comment (Phaeton) [inline plane-alert-db.csv:14196]: I'd set this is Historic

**Review by Phaeton — CHANGES_REQUESTED** [review]

File: plane-alert-db.csv (hunk 1 above)
Comment (lingawakad) [reply plane-alert-db.csv:14200]: that's what the other DA-42s were tagged as, but i'd be fine with it going in distintive or taken out if it isn't actually distinctive

File: plane-alert-db.csv (hunk 2 above)
Comment (lingawakad) [reply plane-alert-db.csv:14194]: ah good call

File: plane-alert-db.csv (hunk 1 above)
Comment (Phaeton) [reply plane-alert-db.csv:14200]: It looks like a standard DA-42 NG, used by a small marketing firm. If its something you want to track, go ahead and put it in, but I'd tag it as As Seen on TV or something similar.

Comment (lingawakad) [reply plane-alert-db.csv:14200]: no worries, i'll take it out

**Review by Phaeton — APPROVED** [review]

**Pull request merged**